packages/ui/src/1_atoms/Paragraph/Paragraph.module.css

```diff
@@ -0,0 +1,42 @@
+.paragraph {
```

Comment:
Btw where did you get info about all of the variations? I was looking at the Figma design and I'd expect to have there all of the variations listed but instead, I found a one example of the Paragraph.

Reply:
they are in Jira story, @pietro-maximoff please request them to be added to the figma

Reply:
updated task desc and asked Gilad to recheck it

Reply:
synced with Gilad, added updates to the story + updated link to design